api/src/utils/user-utils.js

```diff
@@ -24,6 +24,13 @@ const nonEditableFields = [
   'role',
   'status',
 ];
+// Fields and their regex for validation
+const validationFields = {
+  email: /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/,
+  phone: /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/,
```

Comment:
This query allows for phone numbers in a variety of formats (978754-5343, 978.754-5343, etc). Should we instead decide to enforce a 10 digit sequence and splice out delimiters like spaces, hyphens, and `.`'s on the frontend? This way we can also enforce a standardized format as well as one that better guarantees legitimate phone numbers.

e.g. User enters `555-545-7565` on the frontend and it gets sent to the API as `5555457565` and validated as such